tests/kernel/sched/schedule_api/src/test_sched_timeslice_reset.c

```diff
@@ -14,7 +14,17 @@
 BUILD_ASSERT(NUM_THREAD <= MAX_NUM_THREAD);
 
 /* slice size in millisecond */
+#if defined(CONFIG_SOC_FAMILY_STM32)
+/*
+ * Tests may fail with low system frequency.
+ * SLICE_SIZE = 600 allows to pass tests with frequency greater than 32MHZ.
+ * But this value is not valid for all chips: buildkite detected issue on
+ * several platforms like mps2_an521, native_posix_64, nrf52_bsim ...
```

Comment:
I'm not sure I understand this comment.
You are saying a SLICE_SIZE of 600 causes these other platforms to fail?

Reply:
Yes